uber/site_sections/registration.py

```diff
@@ -41,6 +41,12 @@ def index(self, session, message='', page='1', search_text='', uploaded_id='', o
             count = total_count
 
         attendees = attendees.order(order)
+
+        groups = set()
+        for a in session.query(Attendee).filter(Attendee.first_name == '', Attendee.group_id != None) \
+                                        .options(joinedload(Attendee.group)).all():
+            groups.add((a.group.id, a.group.name+" ("+a.group.leader.full_name+")" or 'BLANK'))
```

Comment:
Ok, I fixed that problem, but what's happening now is that it only shows groups with unclaimed badges, but that means that when you check someone in, the dropdown doesn't contain the group that the person already belongs to... which is probably not what we want, I don't think.